RZUtils/Components/RZSingleChildContainerViewController/RZSingleChildContainerViewController.m

```diff
@@ -126,7 +126,7 @@ - (UIViewController *)childViewControllerForStatusBarHidden
     return self.currentContentViewController;
 }
 
-- (NSUInteger)supportedInterfaceOrientations
```

Comment:
Won't this throw a type mismatch warning in Xcode 6? Wondering if a compiler directive is in order here.